src/components/sbb-footer/sbb-footer.tsx

```diff
@@ -7,67 +7,33 @@ import { InterfaceFooterAttributes } from './sbb-footer.custom';
   tag: 'sbb-footer',
 })
 export class SbbFooter {
-  /** Footer appearance */
-  @Prop() public appearance?: InterfaceFooterAttributes['appearance'] = 'primary';
+  /**
+   * Negative coloring variant flag
+   */
+  @Prop() public negative: boolean;
```

Comment:
Maybe set a default value to false can help; also consider using reflect: true. (see button PR: https://github.com/lyne-design-system/lyne-components/pull/1328/files#diff-11fe76111422e9503ceb8218483386e577bf6f23af6d66c4199c721d42756970)